tests/framework/framework.go

```diff
@@ -96,6 +96,23 @@ func init() {
 	master = flag.String("master", "", "master url:port")
 }
 
+// NewFrameworkOrDie calls NewFramework and handles errors by calling Fail. Config is optional, but
+// if passed there can only be one.
+func NewFrameworkOrDie(prefix string, config ...Config) *Framework {
```

Comment:
Why is config variadic?

Reply:
So that it becomes an _optional_ parameter.